site/src/app/Header.tsx

```diff
@@ -94,25 +94,44 @@ function Header(props: Props): JSX.Element {
                     </div>
 
                     <div className="absolute bottom-0 flex w-full justify-center">
-                        <input
-                            type="search"
-                            placeholder="Filter applications..."
-                            value={props.filters.textSearch}
-                            onChange={handleTextSearchChange}
-                            aria-label="Filter applications"
-                            className="relative top-5 mx-4 h-14 w-full max-w-5xl p-3 pl-3 outline-none drop-shadow-md lg:mx-0 lg:w-2/3"
-                        />
+                        <div className="relative flex w-2/3 relative top-5 mx-4 h-14 max-w-5xl">
```

Comment:
`relative` is twice in that class string.

Reply:
For medium widths the filter bar is now too narrow. (It is ok for mobile or wider widths)

Before:
![image](https://github.com/nrfconnect/ncs-app-index/assets/260705/bb5b54cb-f319-4ae4-ad0f-7de21bae3828)

Now:
![image](https://github.com/nrfconnect/ncs-app-index/assets/260705/40533faa-8c61-4dc4-965f-390f1c4e47a4)

Reply:
Thanks! Fixed now